modules/security-context-constraints-pod-security-admission.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * authentication/managing-security-context-constraints.adoc
+
+:_content-type: CONCEPT
+[id="security-context-constraints-pod-security-admission_{context}"]
+= Pod security admission on {product-title}
+
+Historically, security context constraints (SCCs) resembled the pod security policies of Kubernetes. Because pod security policies are now deprecated in favor of pod security admission, SCCs were adapted to give you access to pod security admission control features.
+
+{product-title} 4.11 includes a new set of SCCs that enforce pod security in accordance with new pod security admission controls. These SCCs are:
+
+* `restricted-v2`
+* `hostnetwork-v2`
+* `nonroot-v2`
+
+These SCCs correspond to older, similarly labeled SCCs, but with the following enhancements:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
+* `seccompProfile` is set to `runtime/default` by default.
+* `allowPrivilegeEscalation` must be unset or set to `false` in security contexts. 
+
+// TODO: The SCCs from previous versions of {product-title} remain available, although they can cause permission mismatches in some situations.
+
+New clusters use the `restricted-v2` SCC in place of the `restricted` SCC that is available to any authenticated user. Clusters that are upgraded to 4.11 from previous versions allow both `restricted-v2` and `restricted` SCCs. 
+
+The SCC admission controller specifies that a container's `runAsNonRoot` field is set to `true` for containers that run as non-zero users.
+
+These changes allow SCC admission to reach the `restricted` pod security profile to restrict pod creation.
```

Comment:
To be clear, you'd prefer all of the content in `modules/security-context-constraints-pod-security-admission.adoc` to be moved over to #47151? I'm not familiar with this area of the product--if it's not necessary to keep this in the proper docs for a couple of releases or longer, that's fine by me.

Reply:
What I mean here specifically is you can't mention changes unless you specify changes to what.

Are these changes to Kubernetes? Are they to OpenShift 4.11? Are they to 4.10? Are they to 4.12? Why mentioning changes in a stable doc to any of those? Maybe leave this out? Also leave `The SCC admission controller specifies that a container's `runAsNonRoot` field is set to `true` for containers that run as non-zero users.` out because it is just an implementation detail, isn't it?

Reply:
Perhaps all of these would really make a better place in #47151

Reply:
> Perhaps all of these would really rather be a part of #47151

Yeah, I think that's fine given the changes to the big, already existing SCC table.